src/Service/RequestProcessor.cpp

```diff
@@ -160,35 +160,34 @@ bool RequestProcessor::shouldProcessCommittedRequest(const RequestForSession & c
     if (first_pending_request.request->xid == committed_request.request->xid)
     {
         found_in_pending_queue = true;
+        committed_request.create_time = first_pending_request.create_time;
```

Comment:
Why reset the time?

Reply:
`committed_request.create_time` is leader's `steady_clock` for this requests, we should set it to local `steady_clock` for request to calculate request duration.